user-guide/XCI.rst

```diff
@@ -1,7 +1,7 @@
-Phase 2 - XC50 ARM
-------------------
+XCI - Marvell Thunder X2
+------------------------
 
-Cray XC50 cabinet, codenamed XCI, containing 10'000 ARM cores, available since November 2018.
+Cray XC50 cabinet, codenamed XCI, containing 20’992 ARM cores, available since November 2018.
```

Comment:
Should this say something like "initially installed November 2018, expanded September 2020?

Reply:
I feel like the docs should represent the current state, leave the history to a blog post